search/src/App.js

```diff
@@ -15,115 +15,138 @@ class App extends Component {
-				},
-				component: {
-				  padding: 10
-				}
-			  }}
+				app="weedid"
+				url="http://localhost:9200/"
+				theme={{
+					typography: {
+						fontFamily: 'Raleway, Helvetica, sans-serif',
+					},
+					colors: {
+						primaryColor: '#0A0A0A',
+						titleColor: '#E64626',
+					},
+					component: {
+						padding: 10
+					}
+				}}
 			>
-			<div style={{ display: "flex", flexDirection: "row" }}>
-			<div style={{ display: "flex", flexDirection: "column", width: "40%" }}>
-			    <CategorySearch
+				<div style={{ position: "fixed", width: "20rem", overflow: "scroll", height: "100%" }}>
+					{/* <DataSearch
 						componentId="searchbox"
-						dataField={["annotations.category.common_name.keyword","annotations.category.role.keyword","annotations.agcontext.agcontext_name.keyword","annotations.category.species.keyword"]}
-						categoryField={["annotations.category.common_name.keyword","annotations.category.role.keyword","annotations.agcontext.agcontext_name.keyword","annotations.category.species.keyword"]}
+						dataField={["annotations__category__common_name","annotation__category__role.keyword","agcontext__agcontext_name.keyword","annotation__category__species.keyword"]}
 						placeholder="Search for species, role, or agcontext"
 						autoSuggest={true}
-						defaultSuggestions={[{label: "Weed", value: "weed"}]}
-						highlight={false}
-						highlightField={["annotations.category.common_name.keyword","annotations.category.role.keyword","annotations.agcontext.agcontext_name.keyword","annotations.category.species.keyword"]}
+						defaultSuggestions={[
+							{label: "Weed", value: "weed"},
+							{label: "Carrot", value: "carrot"}
+						]}
+						highlight={true}
+						highlightField={["annotations__category__common_name","annotation__category__role.keyword","agcontext__agcontext_name.keyword","annotation__category__species.keyword"]}
 						queryFormat="or"
 						fuzziness={0}
 						debounce={100}
 						react={{
-						  and: ["CategoryFilter", "SearchFilter"]
+						  and: ["searchbox","resslider","agcontextfilter","rolefilter","speciesfilter","grainstextfilter"]
 						}}
 						showFilter={true}
-						filterLabel="Venue filter"
-						URLParams={false}
-				/>
-				<MultiList
+						filterLabel="General filter"
+						URLParams={true}
+				/> */}
+					<MultiList
 						componentId="rolefilter"
 						title="Filter by role"
-						dataField="annotations.category.role.keyword"
+						dataField="annotation__category__role.keyword"
 						sortBy="asc"
-						queryFormat="or"
+						queryFormat="and"
 						selectAllLabel="All roles"
 						showCheckbox={true}
 						showCount={true}
 						showSearch={true}
 						placeholder="Search Role"
 						react={{
-							and: ["searchbox", "SearchFilter"]
+							and: ["searchbox", "resslider", "agcontextfilter", "rolefilter", "speciesfilter", "grainstextfilter"]
 						}}
 						showFilter={true}
 						filterLabel="Role"
-						URLParams={false}
+						URLParams={true}
```

Comment:
ideally we factor this and queryFormat and other constants out too. I think we can use [JSX spread attributes](https://zhenyong.github.io/react/docs/jsx-spread.html#spread-attributes)